### PR TITLE
+ add an even more baseline integration test with concent disabled for both nodes

### PR DESCRIPTION
**Pull request by shadeofblue**
[ please merge it after approval ]

### CODE REVIEW

File: scripts/concent_integration_tests/params.py [hunk 1]
```diff
@@ -43,3 +43,9 @@ def get_datadir(role: str):
 REQUESTOR_ARGS_DEBUG = REQUESTOR_ARGS + [
     '--log-level', 'DEBUG',
 ]
+
+REQUESTOR_ARGS_NO_CONCENT = list(REQUESTOR_ARGS_DEBUG)
+REQUESTOR_ARGS_NO_CONCENT[1] = 'disabled'
```
Comment (ktomala) [inline scripts/concent_integration_tests/params.py:48]: I don't like this. This is prone to errors when ordering will change. Would suggest doing ordered dict and then creating a list out of it.

File: scripts/concent_integration_tests/params.py [hunk 2]
```diff
@@ -43,3 +43,9 @@ def get_datadir(role: str):
...
+
+PROVIDER_ARGS_NO_CONCENT = list(PROVIDER_ARGS_DEBUG)
+PROVIDER_ARGS_NO_CONCENT[1] = 'disabled'
```
Comment (ktomala) [inline scripts/concent_integration_tests/params.py:51]: Same here

File: scripts/concent_integration_tests/tests/test_concent_disabled.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+
+class RegularRun(NodeTestPlaybook):
+    provider_node_script = 'provider/no_concent'
+    requestor_node_script = 'requestor/no_concent'
+
+
+playbook = RegularRun.start()
+if playbook.exit_code:
+    print("exit code", playbook.exit_code)
```
Comment (ktomala) [inline scripts/concent_integration_tests/tests/test_concent_disabled.py:14]: Should change to logging I think.

**Review by ktomala — APPROVED** [review]

File: scripts/concent_integration_tests/params.py (hunk 1 above)
Comment (shadeofblue) [reply scripts/concent_integration_tests/params.py:48]: it doesn't need to be an _ordered_ dict, actually ... but yeah, a good idea!

**Pull request merged**